src/Language/Mulang/Parsers/Python.hs

```diff
@@ -226,27 +226,27 @@ muOpReference (In _)                 = M.Reference "in"
 muOpReference (Is _)                 = M.Reference "is"
 muOpReference (IsNot _)              = M.Reference "is not"
 muOpReference (NotIn _)              = M.Reference "not in"
-muOpReference (BinaryOr _)           = M.Primitive O.Or
+muOpReference (BinaryOr _)           = M.Primitive O.BitwiseOr
 muOpReference (Xor _)                = M.Reference "^"
-muOpReference (BinaryAnd _)          = M.Primitive O.And
-muOpReference (ShiftLeft _)          = M.Reference "<<"
-muOpReference (ShiftRight _)         = M.Reference ">>"
+muOpReference (BinaryAnd _)          = M.Primitive O.BitwiseAnd
+muOpReference (ShiftLeft _)          = M.Primitive O.BitwiseLeftShift
+muOpReference (ShiftRight _)         = M.Primitive O.BitwiseRightShift
 muOpReference (Multiply _)           = M.Primitive O.Multiply
 muOpReference (Plus _)               = M.Primitive O.Plus
 muOpReference (Minus _)              = M.Primitive O.Minus
 muOpReference (Divide _)             = M.Primitive O.Divide
 muOpReference (FloorDivide _)        = M.Reference "//"
 muOpReference (Invert _)             = M.Reference "~"
```

Comment:
C has the same operator (Bitwise Not) but we didn't add support for this. Maybe we can do it.